scripts/ci/submit-codecov-coverage.sh

```diff
@@ -19,15 +19,11 @@
 # If we're on Travis, then we need to manually check that the build succeeded.
 if [[ "${USER}" == "runner" || ${TRAVIS_TEST_RESULT} -eq 0 ]]; then
     # 1. Install codecov dependencies
-    # NOTE: We need eventlet installed so coverage can be correctly combined. This is needed because we are covering code which utilizes eventlet.
-    # Without eventlet being available to the coverage command it will fail with "Couldn't trace with concurrency=eventlet, the module isn't installed."
-    pip install eventlet
-    # NOTE: codecov only supports coverage==4.5.2
-    pip install 'coverage<5.0'
```

Comment:
With removing `coverage` (https://github.com/StackStorm/st2/pull/6040/commits/9f27017719652496cc43f717e04deaa48d118142) we get the following error at a later stage:

https://github.com/StackStorm/st2/actions/runs/6603759812/job/17937403741?pr=6040#step:17:38

```
warning - 2023-10-22 12:48:46,429 -- coverage.py is not installed or can't be found.
info - 2023-10-22 12:48:46,522 -- Found 0 coverage files to upload
Error: No coverage reports found. Please make sure you're generating reports successfully.
Error: Process completed with exit code 1.
```

so it's indeed needed

Reply:
Will put it back one sec here.

Reply:
I apologize about that. That should do it for ya.

Reply:
No worries, and thanks for your work! 👍